packages/labs/ssr/src/lib/util/parse5-utils.ts

```diff
@@ -4,11 +4,16 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {DefaultTreeAdapterMap} from 'parse5';
-import {traverse, replaceWith, isElementNode} from '@parse5/tools';
+import {
+  traverse,
+  replaceWith,
+  isElementNode,
+  Element as p5Element,
```

Comment:
All type names should start with a capital letter. And I'm personally a fan of longer, more readable, names. I would go for `Parse5Element`, although in these libraries the DOM types aren't loaded and just `Element` and `Node`, etc would be fine with me too.

Reply:
sure no worries, ill just revert to the normal names